src/connectors/dynamic-widgets/__tests__/connectDynamicWidgets-test.ts

```diff
@@ -590,11 +627,54 @@ describe('connectDynamicWidgets', () => {
         widgetParams,
       });
     });
+
+    it('warns when facets is unset and more than 20 items are returned from attributesToDisplay', () => {
+      const dynamicWidgets = connectDynamicWidgets(() => {})({
+        transformItems() {
+          return Array.from({ length: 21 }, (_, i) => `test${i}`);
```

Comment:
`Array.from(Array(21), ...)`  would probably be more readable.

Reply:
why? that's what we are doing in other cases too, see eg. the range helper :)